apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -1,21 +1,21 @@
 import {
-	type ColumnDef,
 	flexRender,
 	getCoreRowModel,
+	type TableColumn,
 	useReactTable,
-} from "@tanstack/react-table";
+} from "~/libs/types/types.js";
```

Comment:
We want to keep only TableColumn in common types, all others are not supposed to be used elsewhere except the Table component itself.

so imports should look like this:
```
import {
	flexRender,
	getCoreRowModel,
	useReactTable,
} from "@tanstack/react-table";
import { type TableColumn } from "~/libs/types/types.js";
```

Reply:
Ok, I'll fix it now